src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -138,6 +147,7 @@ export class VisualizeEmbeddable
       VisualizeByReferenceInput
     >,
     savedVisualizationsLoader?: SavedVisualizationsLoader,
+    savedAugmentVisLoader?: SavedAugmentVisLoader,
```

Comment:
Nit: We can declare this private here which will combine lines `138` and `173` below
`private savedAugmentVisLoader?: SavedAugmentVisLoader`

Reply:
Not sure what you mean here. It is declared private on 138, and line 150 is just specifying an arg in the constructor. I was following the patterns for the rest of the `visualize_embeddable` private fields, and am consistent with the existing `savedVisualizationsLoader`